pbrd/pbr_vty.c

```diff
@@ -1865,13 +1865,14 @@ DEFPY(show_pbr_nexthop_group,
       JSON_STR)
 {
 	json_object *j = NULL;
+	uint16_t pbr_nhg_count = 0;
 
 	if (json)
-		j = json_object_new_array();
+		j = json_object_new_object();
 
 	if (j) {
-		pbr_nht_json_nexthop_group(j, word);
-
+		pbr_nht_json_nexthop_group(j, word, &pbr_nhg_count);
+		json_object_int_add(j, "nhgCount", pbr_nhg_count);
```

Comment:
this is a bit confusing: first, it seems like the count of objects in the collection (whether an array or a dict) should be pretty easy to find? and second, it's confusing to have a dict where some keys are the string form of nhg ids, and their objects are nhg objects, but some other keys are ... entirely different, and key entirely different objects.